modules/expo-bluesky-video-player/ios/ExpoBlueskyVideoPlayerView.swift

```diff
@@ -0,0 +1,65 @@
+import ExpoModulesCore
+
+public class ExpoBlueskyVideoPlayerView: ExpoView, AVPlayerViewControllerDelegate {
+  let onPlayerStateChange = EventDispatcher()
+  
+  private var controller: PlayerController
+  
+  var source: String? = nil
+  var autoplay = true
+  var isPlaying: Bool
+  
+  public override var bounds: CGRect {
+    didSet {
+      controller.setFrame(rect: bounds)
+    }
+  }
+  
+  public required init(appContext: AppContext? = nil) {
+    self.isPlaying = self.autoplay
+    self.controller = PlayerController()
+    super.init(appContext: appContext)
+    self.clipsToBounds = true
+    
+    self.controller.setFrame(rect: bounds)
+    self.addSubview(self.controller.view)
+  }
```

Comment:
This is the main native view. Here, we create a `AVPlayerViewController` that will host the actual `AVPlayer`. It's called `PlayerController` because we are subclassing `AVPlayerViewController` to add some useful functionality.